### PR TITLE
refactor(console): update pro and beta tags

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

- refactor `<ProTag>` to `<FeatureTag>` which can also render a beta tag
- simplify the logic of tag rendering, remove unnecessary checks
- add comments

## Testing

image
image
image

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [x] necessary TSDoc comments

### CODE REVIEW

File: packages/console/src/components/FeatureTag/index.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ * Use as a paid plan tag:
+ *
+ * ```tsx
+ * // In a production tenant, the tag will be visible when there's no access to the feature
+ * <FeatureTag for="upsell" isVisible={noAccessToFeature} plan="pro" />
+ *
+ * // In a dev tenant, the tag will always be visible even if `isVisible` is `false`
+ * <FeatureTag for="upsell" isVisible={false} plan="pro" />
+ *
+ * // Fro conditionally rendering the tag, usually in an iteration on a list which contains
```
Comment (xiaoyijun) [inline packages/console/src/components/FeatureTag/index.tsx:57]:
```suggestion
 * // For conditionally rendering the tag, usually in an iteration on a list which contains
```

File: packages/console/src/components/InlineUpsell/index.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import { contactEmailLink } from '@/consts';
+import InlineNotification from '@/ds-components/InlineNotification';
+import TextLink from '@/ds-components/TextLink';
+import useTenantPathname from '@/hooks/use-tenant-pathname';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  className?: string;
+  for: NormalizeKeyPaths<LocalePhrase['translation']['admin_console']['upsell']['paywall']>;
```
Comment (xiaoyijun) [inline packages/console/src/components/InlineUpsell/index.tsx:15]:
```suggestion
  for: TFuncKey<'translation', 'admin_console.upsell.paywall'>;
```

**Review by xiaoyijun — APPROVED** [review]
Cool, LGTM

File: packages/console/src/components/CreateConnectorForm/Footer/index.tsx [hunk 3]
```diff
@@ -71,6 +71,7 @@ function Footer({
         <QuotaGuardFooter>
           <Trans
             components={{
+              b: <b />,
```
Comment (charIeszhao) [inline packages/console/src/components/CreateConnectorForm/Footer/index.tsx:74]: nit: If I remember correctly, the i18next can render `<strong />` properly without writing this `<Trans />` rules. I've been using this on the official website

Comment (gao-sun) [reply packages/console/src/components/CreateConnectorForm/Footer/index.tsx:74]: let me take a look and update to `<strong>`

**Review by wangsijie — APPROVED** [review]

File: packages/console/src/components/CreateConnectorForm/Footer/index.tsx (hunk 3 above)
Comment (gao-sun) [reply packages/console/src/components/CreateConnectorForm/Footer/index.tsx]: works

**Pull request merged**